ci-operator/step-registry/upi/deprovision/vsphere/upi-deprovision-vsphere-commands.sh

```diff
@@ -95,4 +95,3 @@ else
   pwsh -file powercli/upi-destroy.ps1 &
   wait "$!"
 fi
-
```

Comment:
How about to revert this useless change? so that PR can be merged without approving from vsphere-approvers.